content/en/Getting started/what-to-expect.md

```diff
@@ -40,17 +40,19 @@ results. Here's what you can expect:
 1. Once the pentest is complete, we move your pentest from _Planned_ to _Remediation_.
 1. You can start assessing all discovered vulnerabilities. In the Cobalt app, navigate
    to **Pentests**. Select your pentest, and navigate to the **Findings** tab.
-   - Scroll down until you see **Activity**. Depending on your assessment, you can
+   - Find the **Status** list. Depending on your assessment, you can
```

Comment:
Changing this to State vs Status.

Reply:
@molfinn Changed _status_ to _state_ across the guide.